tests/components/konnected/test_config_flow.py

```diff
@@ -1056,8 +1111,9 @@ def mock_constructor(host, port, websession):
     assert schema["momentary"] == 50
     assert schema["pause"] == 100
     assert schema["repeat"] == 4
+    assert schema["more_states"] == "Yes"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5266431)